src/js/providers/html5.js

```diff
@@ -57,18 +57,22 @@ function _removeListeners(eventsHash, videoTag) {
 }
 
 function VideoProvider(_playerId, _playerConfig, mediaElement) {
+    const _this = this;
+
     // Current media state
-    this.state = STATE_IDLE;
+    _this.state = STATE_IDLE;
 
     // Are we buffering due to seek, or due to playback?
-    this.seeking = false;
+    _this.seeking = false;
 
     // Value of mediaElement.currentTime on last "timeupdate" used for decode error retry workaround
-    this.currentTime = -1;
+    _this.currentTime = -1;
 
     // Attempt to reload video on error
-    this.retries = 0;
-    this.maxRetries = 3;
+    _this.retries = 0;
+    _this.maxRetries = 3;
+
+    _this.loadAndParseHlsMetadata = _playerConfig.loadAndParseHlsMetadata || true;
```

Comment:
wouldn't this always return true even if `loadAndParseHlsMetadata` is false?

Reply:
woops! Thanks.